### DISCUSSION TITLE
Use bundled pandoc without quarto?

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#510] [Q&A] [answered]
If we install quarto CLI on Windows, the directories where quarto and pandoc exist are added to the PATH env var, so the `pandoc` command works.
On Linux (deb package), the symbolic link of quarto is created, but not for pandoc, so the `pandoc` command does not work (Of course, we can run pandoc directly by specifying the path to the bundled pandoc executable.).

It might be useful if the `pandoc` command also works on Linux like on Windows.

**Comment by jjallaire** [comment, marked as answer]
We think that adding `pandoc` to the `PATH` is a bit too aggressive (as users may have their own versions of pandoc already on the `PATH`). We do however have a shortcut for accessing the bundled version of pandoc:

```bash
quarto pandoc <...args>
```

That will pass straight through to pandoc (arbitrary pandoc args are supported)

**Reply by eitsupi** [reply]
That make sense, thanks!

However, I believe Pandoc is currently added to the PATH on Windows.
(Execute `where.exe pandoc` to find the pandoc executable bundled with quarto.)

**Reply by jjallaire** [reply]
Yes, you are correct, we should be moving the ancillary binaries into their own subdirectory so they don't pollute the `PATH` on Windows. cc @dragonstyle

**Reply by dragonstyle** [reply]
Will do.
